doc/visual-programming/source/widgets/unsupervised/distancetransformation.rst

```diff
@@ -0,0 +1,57 @@
+Distance Transformation
+=======================
+
+.. figure:: icons/distance-transformation.png
+
+Transforms distances in a data set. 
+
+Signals
+-------
+
+**Inputs**:
+
+-  **Distances**
+
+A distance matrix
+
+**Outputs**:
+
+-  **Distances**
+
+A distance matrix
+
+Description
+-----------
+
+The **Distances Transformation** widget is used for the normalization and inversion of distance matrices. The normalization of data is necessary to bring all the variables into proportion with one another. 
+
+.. figure:: images/DistanceTransformation-stamped.png 
+
+1. Choose the type of `Normalization <https://en.wikipedia.org/wiki/Database_normalization>`_:
```

Comment:
Wrong link here. Normalization of databases is somewhat different than our [normalization](https://en.wikipedia.org/wiki/Normalization_(statistics)).